### ISSUE TITLE
Spanish translation

### ISSUE THREAD

**Issue by medied** [DemocracyEarth/paper#5]
Was talking to my mom about DEF the other day and realized how different it felt to explain it in spanish 🤔 . It would be nice to have the whitepaper translated to different languages for people who are more comfortable in a language other than english. I bet it would help in spreading the word too. 

So was thinking about starting the spanish translation but thought I would check in first. @santisiri what do you think? This is not urgent of course and we should probably get the english one ready first

**Comment by santisiri** [comment]
I think it's an amaing idea and would love to have this in spanish. We can wait until the first draft is completed. But if you want to begin already, I'm glad to help as well.

Not sure what's the best convention but we could add a README_ES.mediawiki file for the translation. Feel free to start this anytime.

**Comment by SFSandra** [comment]
This is such a fantastic idea.  We can have a version in Portugese too, I'm sure Agatha or Paula would be happy to do the translation.

**Comment by santisiri** [comment]
Portuguese would be sweet. We have a big community in Brazil.

**Comment by medied** [comment]
cc'ing @paulamlb 

Anyone who wants to translate to a different language should feel free to work under the [`translations`](https://github.com/DemocracyEarth/paper/tree/master/translations) directory :)

**Comment by paulamlb** [comment]
Amazing! Will most definitely translate to Portuguese. As there is still a lot to be worked on, I will do that next week. Also I don't know if everybody does that, but I often translate things and my trick is to google translate it all and then go over everything (carefully) afterwards. Saves A LOT of time and works well.

**Comment by SFSandra** [comment]
Paula that is a good plan!

**Comment by medied** [comment]
@paulamlb yes! That's what I do too, does save a lot of time :)

**Comment by medied** [comment]
Just to have a minimum of organization and depending on how many people jump to translations, am thinking we can use Github projects to coordinate (like the the one below) and Slack channels (see #habla-español): 

https://github.com/DemocracyEarth/paper/projects/1

@paulamlb what do you think? Is it worth to have one for Portuguese? Or Chinese or whatever language. 

This is of course if there's a lot of people interested in translating, otherwise a more simple approach might work just fine

**Comment by medied** [comment]
Just flagging FYI I just realized projects don't really work well to coordinate in this case: Github requires you to be a member of DEF to add cards to the project, oh well

**Comment by santisiri** [comment]
@paulamlb @SFSandra official membership to our github repo invitation sent.

**Comment by nelruk** [comment]
Hello @santisiri @medied question: I finished to translate a portion and when I want to PR says I don't have the permission to do it. I'm trying to put from my branch to spanish-translation.

Could you help me with this?

**Comment by bhack** [comment]
How the translation are maintained in sync?

**Comment by bhack** [comment]
I don't know if we will never touch the document but when it is online there is https://m.mediawiki.org/wiki/Extension:Translate

**Comment by medied** [comment]
You mean when the original english document gets updated? Right now translations have to be manually updated to stay in sync. So thanks for that extension! Might come in handy for any future automation.

**Comment by nelruk** [comment]
The first thing we need to finish is fully translated the document, make proper editions and then automation, which could be a medium-term objective.

**Comment by bhack** [comment]
Yes but you need also to check every translation if you touch the english content. How do you notify the translations contributors on each PR that you merge on master over the english text? Cause I suppose that the Master document is in english right?

**Comment by medied** [comment]
Yea exactly this could be a source confusion (see #204 for reference). I guess as long as we are doing it manually, every time `master` is updated we need to cascade that to every translation. An automatic solution would be very nice eventually

**Comment by bhack** [comment]
With the extension:
>Each unit is automatically tracked for changes, and translators immediately see what needs updating on a specific page or throughout the wiki.

**Comment by domi41** [comment]
There's a way to automate, using github :

```
For each new commit :
If commit does not concern root's README.mediawiki, stop here
Find line numbers that changed
Parse mediawiki file to cut it into chapters (regex FTW)
Find the chapters the edited lines correspond to
Find contributors to those chapters in the other languages
Create an issue, with proper tags, description and assignees
Maybe even create an associated card in a project (github API is still experimental here)
```

If the chapter structure changes, it should be easy enough to compare with the translations' chapter structure and yell accordingly in a special case.

This can be hacked rather quickly with few lines of python. But we'd need a server to host the bot, so that github can send its events to it.

**Comment by domi41** [comment]
Also see #211

**Comment by marcoht** [comment]
Hello everyone. I'm starting to review the portuguese translation and realized that there are a lot figures to be translated. Do the editable files available? If don't, I'll try to write the translations over (means to add portuguese text boxes over the english text).

**Comment by SFSandra** [comment]
Hi, thanks for the question, all the figures appear in this file paper/images/dearth-paper-assets.sketch

**Comment by SFSandra** [comment]
Hello! The figures can be found in this [file](paper/images/dearth-paper-assets.sketch)

**Comment by SFSandra** [comment]
hi @marcoht will [this work](https://www.dropbox.com/sh/u0iy0g79lm4u68y/AAAic_Xe1U25br_9oZlFy7QKa?dl=0)

**Comment by marcoht** [comment]
It worked! Thanks!

**Comment by marcoht** [comment]
@SFSandra and @paulamlb I have finished most images' translation. How can I upload the files? Can I upload directly in the same directory where you've uploaded the english ones? Also, I've created ".xcf" (GIMP) editable files, which can be used to any translation easily.
The schematic images of "dripping", PoI and smart contracts are still missing translation to portuguese. I'm working on them right now.

**Comment by SFSandra** [comment]
Hi @marcoht wow that is great news, you work fast! honestly I am not sure if you can upload to the repository - I *think* that you need to be invited to have that level of permission. I will check on that - otherwise, if you have a dropbox file you can share that way (sandra@democracy.earth), and I will upload to the repo .

**Comment by marcoht** [comment]
Hi @SFSandra ! Thanks... I've uploaded some other files today there in the Dropbox folder. Now it is missing just one last.
@paulamlb only one file missing now.
